### PR TITLE
kvm: fix NPE while import KVM VMs from other hosts

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes NPE while import VMs from kvm host.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by weizhouapache** [comment]
this issue was found by @kiranchavala  
refer to https://github.com/apache/cloudstack/issues/8704#issuecomment-1963762570

File: plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/resource/LibvirtDomainXMLParserTest.java [hunk 1]
```diff
@@ -259,4 +259,126 @@ public void testDomainXMLParser() {
         assertEquals(WatchDogDef.WatchDogModel.I6300ESB, watchDogs.get(0).getModel());
         assertEquals(WatchDogDef.WatchDogAction.RESET, watchDogs.get(0).getAction());
     }
+
+    @Test
+    public void testDomainXMLParser2() {
```
Comment (DaanHoogland) [inline plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/resource/LibvirtDomainXMLParserTest.java:264]: how about a more explicit name?
```suggestion
    public void testDomainXMLParserWithoutModelName() {
```
not claiming this name is good or the best by any means.

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Comment by weizhouapache** [comment]
thanks @DaanHoogland 
suggestion committed.

**Review by sureshanaparti — APPROVED** [review]
code lgtm

**Comment by sureshanaparti** [comment]
@blueorangutan test

**Review by vishesh92 — APPROVED** [review]
clgtm

**Pull request merged**